test/test_cli.rb

```diff
@@ -26,9 +26,8 @@ def wait_booted
   end
 
   def teardown
-    File.unlink @tmp_path if File.exist? @tmp_path
-    File.unlink @tmp_path2 if File.exist? @tmp_path2
-
+    File.unlink @tmp_path  if @tmp_path && File.socket?(@tmp_path)
+    File.unlink @tmp_path2 if @tmp_path && File.socket?(@tmp_path2)
```

Comment:
Did you mean...
```ruby
File.unlink @tmp_path2 if @tmp_path2 && File.socket?(@tmp_path2)
```
?

Reply:
Thanks. Long day, stupid mistake.  I'll fix with a push.